src/grappler/ngraph_optimizer.cc

```diff
@@ -40,8 +40,9 @@ namespace ngraph_bridge {
 Status NgraphOptimizer::Optimize(tensorflow::grappler::Cluster* cluster,
                                  const tensorflow::grappler::GrapplerItem& item,
                                  GraphDef* output) {
-  NGRAPH_VLOG(5) << "Here at NgraphOptimizer ";
-  NGRAPH_VLOG(5) << "NgraphOptimizer : grappler item id " << item.id;
+  NGRAPH_VLOG(0) << "[NGTF-OPTIMIZER] Here at NgraphOptimizer ";
```

Comment:
remove line #43 and keep just #44? 
incase you want to keep both for different log levels, then we can remove "Here at NgraphOptimizer" and just keep [NGTF-OPTIMIZER]  right?

Reply:
Using [NGTF-OPTIMIZER] as a TAG with every log and I want to keep both because LOG(0) would show up anyway, making it easier to figure out that the grappler is being used.